### DISCUSSION TITLE
Alpha-spectrometer for airborne radon detection

### DISCUSSION THREAD

**Discussion by bernard-bebo** [ozel/DIY_particle_detector#31] [Q&A]
Hello, I’m interested in doing a DIY Arduino based IoT project to continuously read the Radon level in my house. After few research on the internet, I have found the DIY Particle Detector/Alpha-spectrometer very interesting and before ordering parts, I would have some questions to see if I’m in the right direction: 

1) Basically, using the variant version (BPX61 PIN Diode) do you think the Alpha-spectrometer would be sensitive enough to detect airborne radon level? (I understand it would have to be placed in shielded and dark box with a ventilation chamber so the radon particles can hit the sensor). 

2) Also, the output of the Alpha-spectrometer are audio pulses, can these pulses (or CPM) be converted in radon level, for instance Bq/m^3? 

That would be great if someone could share thoughts about this?
Thanks a lot!

**Comment by ozel** [comment]
Radon detection in houses is a great idea and also something on my own todo list. In principle, the alpha-spectrometer variant of this project is a perfect basis. If no other radioactive source is in front of the detector diode, every pulse with an energy larger than about 1 MeV must stem either from a radon decay itself or one of its decay products (various polonium isotopes).
You can see a typical energy spectrum in figure 10 of the underlying [article](https://www.mdpi.com/1424-8220/19/19/4264/htm) about my detectors. This measurement was taken from a radium painted dial, from airborne radon you would just be missing the leftmost Ra226 peak.

Only using the CPM information or number of pulses is not enough, because cosmic radiation and gamma radiation will also make a pulse every now and then in the diodes. Have a look at the [wiki](https://github.com/ozel/DIY_particle_detector/wiki/Energy-Spectra) to understand how the pulse size is converted into an energy value. The article linked above also explains more in detail how this energy calibration with reference alpha soruces was done.

So basically, you just look for large pulses (above 1 MeV) in the detector output and can savely assume that they must come from airborne radon alpha decays. From the known volume of the chamber where the diode is measuring inside you can then deduce the radioactivity volume due to radon. The keyphrase I would look for in the literature is _radon diffusion chamber_, that should provide you with some ideas about how to design this measurement volume. Some commercial smarthome radon sensors use exactly this principle in combination with an energy sensitive semiconductor detector like the BPX61 diode.

I would not use a fan and rather work with a defined volume and openings. The measurement will not be a quick one, since the diode is small, it will take several hours up to days for recording a meaningful average value from the pulse rate.
Adding a fan would speed it up but should also introduce other difficulties like air turbulence and maybe unexpected flows of air that are difficult to take into account. 

Please let me know how it goes! I will not have time to try it myself soon but will be glad to assist you in trying it.

**Reply by bernard-bebo** [reply]
Hello! Thank you very much for your feedback and relevant information. I will definitely start by building the DIY spectrometer and get familiar with the tools and share some results. I can’t wait to get started, but I also have a limited time for this project.

I was wondering if you could share your electronic schematic (kicad?), this may be very useful for some of us who want to first build the circuit on a prototype board instead of ordering PCB? 

Best regards,

**Reply by ozel** [reply]
the design files are of course all in the [hardware directory](https://github.com/ozel/DIY_particle_detector/tree/master/hardware/V1.2), would be difficult to claim this is an open science hardware project otherwise ;-)

regarding prototyping on a breadboard: this usually does not work as this circuit is way to sensitive to parasitic capacitances and inductances occurring in all the breadboard connections! The boards only cost a few bucks, even if you order just two or three of them. Checkout the Aisler link on the Kitspace project page if you are in Europe or go with PCBWay/JLPCB if you can wait some days longer: https://kitspace.org/boards/github.com/ozel/diy_particle_detector/alpha-spectrometer/

**Reply by bernard-bebo** [reply]
Hello, thanks for pointing me out about the hardware directory, got the file I wanted (schematic). PCB and sound card (Virtual 5.1 Sound Card 3D) are underway... Working on parts. Regards.
